engine/execution/ingestion/engine.go

```diff
@@ -835,6 +835,7 @@ func (e *Engine) handleCollection(originID flow.Identifier, collection *flow.Col
 	lg := e.log.With().Hex("collection_id", collID[:]).Logger()
 
 	lg.Info().Hex("sender", originID[:]).Msg("handle collection")
+	defer lg.Info().Msg("collection handled")
```

Comment:
Could we log the duration as well. Otherwise it’s not easy to find logs that takes too long to handle collection